File: Java/Week_3-4/src/models/Department.java
package models;

public class Department
{
    public String deptId;
    public String deptName;
    public String building;
    public String headOfDept;

    public Department(String deptId, String deptName, String building, String headOfDept)
    {
        this.deptId     = deptId;
        this.deptName   = deptName;
        this.building   = building;
        this.headOfDept = headOfDept;
    }

    @Override public String toString()
    {
        return deptName + " (" + deptId + ")";
    }
}

File: PythonProject/Week 2/models/department.py
class Department:
    def __init__(self, dept_id, dept_name, dept_head_id, building, budget):
        self.dept_id = int(dept_id)
        self.dept_name = dept_name
        self.dept_head_id = dept_head_id
        self.building = building
        self.budget = float(budget)

    def __repr__(self):
        return f"Department({self.dept_id}, {self.dept_name})"
